Makefile

```
@@ -75,6 +75,10 @@ HELM_MANIFEST_CACHE_CONTROL ?=public, max-age=600
 CONTROLLER_GEN ?=$(GO) run ./vendor/sigs.k8s.io/controller-tools/cmd/controller-gen --
 CRD_PATH ?= pkg/api/scylla/v1/scylla.scylladb.com_scyllaclusters.yaml
 
+SWAGGER ?=$(GO) run ./vendor/github.com/go-swagger/go-swagger/cmd/swagger
+MANAGER_CLIENT_DIR ?=pkg/scyllamanagerclient/gen/
+MANAGER_CLIENT_SRC_DIR ?=pkg/scyllamanagerclient/scyllamanagerclient.json
```

```suggestion
MANAGER_CLIENT_SRC ?=pkg/scyllamanagerclient/scyllamanagerclient.json
```
It's not a directory